CHANGES.rst

```diff
@@ -1,3 +1,43 @@
+1.1.0 (2022-04-04)
+===================
+
+New Features
+------------
+
+Project & API Documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Adding RTD Requirements file and yml
+- Updated JDOX links so they don't produce 404 errors 
+- Updates to About page
+
+
+Web Application
+~~~~~~~~~~~~~~~
+
+- Speed up Webapp loading
+- Striped code for createing mosaic preview images
+- Ignoring unuseful pipeline products
+- Updated feature desciption on EDB page
+- Closed all open sessions to db explicitly
+- Fix/Modify Integration Number slider
+- Switch filesystem and astroquery.mast to point to OPS
+- Updates to BANDIT CI
+- Made plotting limits robust to missing data
+- Update EDB
```

Comment:
Could you make this one a little more explicit? These were updates to the EDB queries?